Models/Product.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace BangazonAPI.Models
+{
+    public class Product
+    {
+        [Key]
+        public int ProductId { get; set; }
+
+        [Required]
+        public decimal Price { get; set; }
+        public string Title { get; set; }
+        public string Description { get; set; }
+        [Required]
+        public int CustomerId { get; set; }
+        public Customer Customer { get; set; }
+        [Required]
+        public int Quantity { get; set; }
+        [Required]
+        public int ProductTypeId { get; set; }
+        public ProductType ProductType { get; set; }
```

Comment:
If you add properties that aren't on the ERD, please reference the ticket number that justifies adding them. (For example, where does it say that we'll need to show the `ProductType` and `Customer` of the `Product`?